### ISSUE TITLE
Prettier main page

### ISSUE THREAD

**Issue by TheEZIC** [07th-expansion-ru/wtc-site#2]

**Comment by Aspirin4k** [comment]
Не кажется, что чтобы сайт был модным, стильным и молодежным; нужно пока не поздно (пока мало кода) поделить бек и фронт - завезти какой-нибудь SPA и использовать скрипты питона чисто как API.. или какой у вас план? )

**Comment by Lenin1917** [comment]
Разрушаешь мой дидовский мир со снежинками на главной странице, зачем так??? ) Ну а если сурьёзно, то действительно следует посмотреть Flask-Rest, вроде там без проблем можно отделить бек очень просто. Особенно с учётом того, что потом я хочу переписать бэк на хацкеле или кложе))

**Comment by Aspirin4k** [comment]
Тут еще зависит от того, какая инфраструктура. И как приложение на сервер деплоится?
Если одна виртуальная машинка, то видимо использовать единый http сервис для отдачи фронта и для самой API (не завезти сногсшибательный и нанотехнологичный SSR). Настроить на урл / отдачу хтмлки с подключенным js бандлом для фронта и уже потом грамотно разруливать статику и запросы api на беке (но это уже сделано, насколько я вижу)..

**Comment by Lenin1917** [comment]
Если честно, сейчес я захожу на серв, потом аттачусь к сессии тмукса и там делаю запуск или отрубаю через ^C :D. Да, немного "неправильно", но пока не парился написанием сыстемди сервисов или докеров. 

Про разделение. Пока внимательно смотрю на FastAPI. Он вроде и простой и к тому же быстрый.

**Comment by Aspirin4k** [comment]
Можно на самом деле попробовать развернуть какой нибудь Nginx и использовать и как реверс прокси для бекенда на питоне, и для отдачи статики. тогда приложения будут максимально изолированы друг от друга. 

но тут уже вопрос в том, насколько хочется упороться. так же зависит от того, как сайт использоваться будет? просто архив-свалка новостей из группы?

**Comment by Lenin1917** [comment]
А ну енжиникс уже и так используется (можешь видеть https у сайте). Пока сайт будет как просто свалка статики.

**Comment by Aspirin4k** [comment]
я могу попробовать верстануть индексную страницу более презентабельно, если с этим нужна помощь. у тебя есть какое-то видение?

**Comment by Lenin1917** [comment]
Да, конечно, виденье есть. К сожалению, сейчас вся разработка координируется в вк, что немного не удобно. https://github.com/07th-expansion-ru/wtc-site/blob/master/TODO#%D1%83%D0%BB%D1%83%D1%87%D1%88%D0%B8%D1%82%D1%8C-%D0%B3%D0%BB%D0%B0%D0%B2%D0%BD%D1%83%D1%8E-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%83 вот тут вот можешь посмотреть саму задачу. Вообще, верстай как считаешь нужным, моет идея у тебя будет лучше чем та, которая сейчас есть у нас.

@TheEZIC можешь сюда выложить ссылку на макет на фигме?

**Comment by Aspirin4k** [comment]
#3
